test/inspector/inspector-helper.js

```diff
@@ -393,7 +393,7 @@ Harness.prototype.addStderrFilter = function(regexp, callback) {
 
 Harness.prototype.assertStillAlive = function() {
   assert.strictEqual(this.running_, true,
-                     'Child died: ' + JSON.stringify(this.result_));
+                     `Child died: ${JSON.stringify(this.result_)}`);
```

Comment:
I'm usually the last one to worry about that sort of stuff - but is it possible this terminology is offensive? If we swapped out other trigger references I think it would be nice to have a code base without dead children. 

It's entirely appropriate in terms of my culture and language background - but I want to check how others feel about it. If someone feels strongly - will open an issue.